### ISSUE TITLE
DisplayObjectContainer.getObjectsUnderPoint does not return containers

### ISSUE THREAD

**Issue by mikkeldamsgaard** [bp74/StageXL#209]
The getObjectsUnderPoint method of DisplayObjectContainer does not add containers to the list as it traverses the children.

It should not add the top-level DisplayContainer (that is the display container, the client code calls getObjectsUnderPoint on), but it should add the Display Containers that are children of the top-level Display Container

**Comment by bp74** [comment]
Thanks for the report, I will take a look at it. Do you know if this is the way how Flash does it too?

**Comment by mikkeldamsgaard** [comment]
Acording to the example on this page 

http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/display/DisplayObjectContainer.html#getObjectsUnderPoint()

the flash implementation atleast returns sprites. The current StageXL implementation does not return sprites.

Also, the change is rather small: Here is what I did to fix it in ths short run.
```
List<DisplayObject> getObjectsUnderPoint(DisplayObjectContainer container, Point<num> point, [
List<DisplayObject> returnList]) {

  var tmpPoint = new Point<num>(0.0, 0.0);

  if (returnList is! List) {
    returnList = new List<DisplayObject>();
  }

  for (int i = 0; i < container.children.length; i++) {
    var child = container.children[i];
    child.parentToLocal(point, tmpPoint);

    if (child.bounds.contains(tmpPoint.x, tmpPoint.y)) {
      returnList.add(child);
    }

    if (child is DisplayObjectContainer) {
      getObjectsUnderPoint(child,tmpPoint, returnList);
    }
  }

  return returnList;
}
```

**Comment by bp74** [comment]
Bump! We should fix this since other users have complained about it too.

**Comment by bp74** [comment]
I did some more testing with Flash and it looks like if a Sprite (the DisplayObjectContainer) is only added to the result if the Graphics field in the Sprite is hit. A DisplayObjectContainer is NOT automatically added to the result if one of the children is hit.

commit: 4e8702727214fd85a2fd5ea93368548f96956075

**Comment by bp74** [comment]
This fix was released with version 0.13.1
